service-front/module/Application/test/Controller/KbvControllerTest.php

```diff
@@ -90,4 +112,16 @@ public function testKbvQuestionsFormRenders(): void
         $this->assertControllerClass('KbvController');
         $this->assertMatchedRouteName('root/id_verify_questions');
     }
+
+    public static function data(): array
```

Comment:
Nit, but if you happen to revisit this: would be good to use a more specific name, like `personTypeDataProvider`